### PR TITLE
Feature/128095 mobile month js

### PR DESCRIPTION
**Pull request by paulmskim**
https://central.tri.be/issues/128095

On hold until https://github.com/moderntribe/the-events-calendar/pull/2576 is merged.

Screencast: https://www.loom.com/share/292e0779e5e646839cf7b93f02a782e0

### CODE REVIEW

**Comment by paulmskim** [comment]
@bordoni @juanfra this is good for CR.

**Comment by bordoni** [comment]
As I noted to you on that PR, we need to make sure all of the Modules that interact with Views need to be by container, so a global state here will create problems. 

I wonder if instead of managing the state in a variable, it would be "simpler" to keep the state tied to the DOM, so when a new view is pulled it it resets.

**Comment by paulmskim** [comment]
That's an interesting solution, will think about how else we could tackle it, but there is always that option. Thanks for the input!

File: src/resources/js/views/month-grid.js [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+	/**
+	 * Handle keydown event to move focused grid cell
+	 *
+	 * @since TBD
+	 *
+	 * @param {Event} event event object
+	 *
+	 * @return {void}
+	 */
+	obj.handleKeydown = function( event ) {
```
Comment (ryanurban) [inline src/resources/js/views/month-grid.js]: Nice

**Pull request merged**